cmd/register/files.go

```diff
@@ -19,7 +20,18 @@ If there are already registered entities with v1 version then the command will f
 ::
 
  bin/flytectl register file  _pb_output/* -d development  -p flytesnacks
+	
+Fast Register will register all the fast serialized protobuf files including tasks, workflows and launchplans with default v1 version. Learn more about fast registration  https://docs.flyte.org/projects/cookbook/en/stable/auto/deployment/workflow/fast_registration.html
+::
+
+ bin/flytectl register file  _pb_output/* -d development  -p flytesnacks  -v v2 -l "s3://dummy/prefix"  --additionalDistributionPath="s3://dummy/fast" 
+	
+Fast Register will fail if you didn't pass additional flags like --additionalDistributionPath flags. Fast register o/p only support work with additional flags
+::
 
+ bin/flytectl register file  _pb_output/* -d development  -p flytesnacks  -v v2 -l "s3://dummy/prefix" 
```

Comment:
Add the error o/p in this example so that the user is aware of kind of error message he would see.